superset-frontend/src/features/alerts/components/NotificationMethod.tsx

```diff
@@ -36,6 +36,12 @@ const StyledNotificationMethod = styled.div`
         border-color: ${({ theme }) => theme.colors.error.base};
       }
     }
+
+    .helper {
+      margin-top: 5px;
```

Comment:
Can we use theme variables for all spacings and font sizes?

Reply:
@kgabryje Thank you for the review! I've refactored styles to use theme variables for spacings and font sizes.

Reply:
@kgabryje Could you please review and resolve the conversation?

Reply:
Sorry for the delay! As a rule we try to avoid fractions of grid units like `gridUnit * 2.5` or `gridUnit * 0.75`. Could you please round those values to whole grid units?

Reply:
Before making adjustments, I'd like confirmation on aligning with our guideline to avoid fractional grid units. Based on our 4px gridUnit definition, here are the proposed changes:

```
- margin-bottom: ${theme.gridUnit * 2.5}px;
+ margin-bottom: ${theme.gridUnit * 2 + 2}px;

- margin-left: ${theme.gridUnit * 2.5}px;
- padding-top: ${theme.gridUnit * 0.75}px;
+ margin-left: ${theme.gridUnit * 2 + 2}px;
+ padding-top: ${theme.gridUnit - 1}px;
```

Please review and confirm so I can proceed accordingly.

Reply:
Not exactly - we use the grid units to enforce consistent spacing between elements and to maintain that consistent spacing in case the grid unit is changed (for example as a result of a user customising the theme). So in other words, we should just use full multiples of grid unit.
Of course that takes away some flexibility, as you can't set a margin of 5, 6 or 7 pixels. You can just pick the closest grid unit (the closest multiple of 4) that looks good

Reply:
Side note - you might notice that there are some components that use fractions of grid units. However, those are old components created before we started enforcing that rule and those haven't been refactored yet. In new code we should stick to the full grid units

Reply:
@kgabryje Thanks for detailed explanation.

 I have done below changes and pushed commit. Please assist with triggering the CI and resolving comment, and bringing up ephemeral env as needed.

```
-      margin-bottom: ${theme.gridUnit * 2.5}px;
+      margin-bottom: ${theme.gridUnit * 2}px;
-        margin-left: ${theme.gridUnit * 2.5}px;
-        padding-top: ${theme.gridUnit * 0.75}px;
+        margin-left: ${theme.gridUnit * 2}px;
+        padding-top: ${theme.gridUnit}px;

```

Reply:
Thank you! Spotted 1 remaining fraction, and we can just use 0px instead of theme.gridUnit * 0 🙂